### ISSUE TITLE
touch on umenu when contents change

### ISSUE THREAD

**Issue by 1offby** [Cycling74/miraweb#109]
If the contents of a umenu in mira.frame are changed dynamically, the new contents don't always load properly. Especially if the changed umenu is touched soon after the change. The incorrect umenu contents don't appear to update until miraweb browser is relaunched:

<pre><code>
----------begin_max5_patcher----------
890.3oc0Xt0aSCCEG+41OEQ4QnTEeIoM7s.wiHzTZpoyPhSkiyXCDe2w2921
M5UsUFLo0SiiyuyEeNGG2eNdT5ht6E8oIuO4SIiF8ywiF4GxMvn30iRaqtut
op2Osz5t1VgxjNIbOi3dieb5zjOLHq+VyCI8hFQsIouqUXtUpVkbqPKvCzHU
h5tAk+onwAUCscClFgwqir3nqqL0tm+FsEWvForhoYSRHzRuHi6DL1zrjOGe
H4Ru4zs3quijCktVK5sFckQ1o1kFORalSjE+XCJqUIUvnHwA6MOzH7pH0Mvu
FO18wjmYziLM4iWun17Y6DzJKOZLimd87dk36Vc7GN+arKjo60mHG1m3YDuS
k68MF08Ikd.mhB7A1lGVKBTRktP5dWvoWSW9sIjK1iiqhk9bd6ZY9QbXx+ZN
rsBbsPsLoZsSb4q1zvBL2GCJmezU6rC57oWyZ6C34C+Pte2kcxjaJ0K3YGyc
KOn2tnRsJcxV4qzRuIQlT2Hpz6OLPOcM9beaZd4wBCyOZFu06Clv0LAXv1Ze
.lwcUZUUqX6M9D4yWbd.1tqfG1t6nIBrSEA7+u4IWWos1mQnuQnpVDbaxw2u
jvy81QXayya+RoQzFeAChU+SrFAMJYQIOJyixhnbVTNOJKiRRF9BHR.RBXR.
TBnR.VB3R.XBHSAY5FaEjofLEjofLEjofLEjofLCjYfLaSX.jYfLCjYfLCjY
fLCj4fLGj4fLeSDFj4fLGj4fLGj4fbNHmCx4fbNHmuYwCjyA4bPNGjyA4BPt
.jK.4BPt.jK1jW.xEfbAHW.xy1lNtSwLFo5NwxapLFsbwfQr8a8wp5XYsq5s
YPz8ELLFe2pllN0pCUg+nI1eam173Ytuog5U1dtmTcmrWhhzv88Mmf3EsS0E
2lJ9lImUWJ5KWWpWy9MudYY+mjh0J0US+hiNhKKpp+1Js8PLKexh3EcHFR30
eysaA49qv0F+PuHHzrbkpyBpwd7zCl+jsiO5g4Oy0SNZr2rbi+XGuuaPWizX
bfijs1zRQuQp76ltyjlGlydismshHmghf077zT1YnI1KfhNmPGb6mkhbwexe
AGpz0B5TNT1KjCcpHW4enmPxt8bY2Iz8wI6Ugsd9qcZ2kyl3uTpBW5aejpEt
FFg4y8iTos0rFaA6fNzH59hvulPZa2RgVMHi8VrZ9Wi+MTRGa.L
-----------end_max5_patcher-----------
</code></pre>

**Comment by fde31** [comment]
Yep, can reproduce. Additionally umenu does seem to handle the situation well when the dropdown is open while the content is changed. Filing as bug.
